src/txadmission.cpp

```diff
@@ -790,6 +790,7 @@ bool ParallelAcceptToMemoryPool(Snapshot &ss,
                     if (debugger)
                     {
                         debugger->AddInvalidReason("Inputs are missing");
+                        return false;
```

Comment:
why not moving `return false` in the outer if then clause and remove it here and in the next if?